src/components/views/TitleForm.js

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { isEqual } from 'lodash';
+import { FormattedMessage } from 'react-intl';
+import {
+  Button,
+  HasCommand,
+  IconButton,
+  Pane,
+  PaneFooter,
+  PaneMenu,
+  Paneset,
+} from '@folio/stripes/components';
+import { TitleManager } from '@folio/stripes/core';
+import stripesFinalForm from '@folio/stripes/final-form';
+import { checkScope } from '@folio/stripes-erm-components';
+
+import TitleCardInfo from '../TitleCard/TitleCardInfo';
+import TitleFormInfo from './TitleFormInfo';
+
+import css from '../styles.css';
+
+class TitleForm extends React.Component {
```

Comment:
Can we change this to be a functional component instead? Its fairly straightforward in this case.